kubeflow/core/prototypes/jupyterhub.jsonnet

```diff
@@ -13,6 +13,8 @@
 // @optionalParam repoName string kubeflow-images-public The repository name for JupyterNotebook.
 // @optionalParam disks string null Comma separated list of Google persistent disks to attach to jupyter environments.
 // @optionalParam gcpSecretName string user-gcp-sa The name of the secret containing service account credentials for GCP
+// @optionalParam notebookUid string 1000 UserId of the host user for minikube local fs mount
```

Comment:
Are we changing the default behavior?  It doesn't look like KubeFormSpawner was setting the UID/GID before but now it is.

Reply:
Yes, it is doing this now only in minikube case. This is to make sure that the locally mounted filesystem is read/writable via Jupyter Notebook

Reply:
updating to nulls.